### PR TITLE
Sync colors & small visual fixes

### PR DESCRIPTION
**Pull request by PotatoParser**

### CODE REVIEW

File: src/sass/app.scss [hunk 1]
```diff
@@ -145,14 +145,15 @@
 .stats-bubbles {
   width: 65%;
   display: inline-block;
-	background-color: #1A5298 !important;
+	background-color: $ieee-blue !important;
```
Comment (encadyma) [inline src/sass/app.scss]: indentation

File: src/sass/app.scss [hunk 2]
```diff
@@ -162,8 +163,12 @@
   width: 50%;
 }
 
+.sub-stats {
+  font-size: 1.25rem;
+}
+
 .ieee-btn {
-	background-color: #004080;
+	background-color: $ieee-blue;
```
Comment (encadyma) [inline src/sass/app.scss]: indentation isn't consistent in file

**Review by encadyma — COMMENTED** [review]
Per Neha's suggestions, are you able to implement a forced white background? We can also try to implement a proper looking dark mode via https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme. 

QA looks good otherwise.

File: src/sass/app.scss (hunk 2 above)
Comment (PotatoParser) [reply src/sass/app.scss]: Will make sure to fix indentations!

**Pull request merged**